File: rafael/controle-de-fluxo/JogoDeDado2.cs
class JogoDeDado2
{
	static void Main() 
	{
		int soma = 0;
		for(int i = 1; i <= 5; i++) 
		{
			System.Random gerador = new System.Random();

			System.Console.WriteLine("Lançamento: " + i);
			int numero = (int)(gerador.NextDouble() * 6 + 1);

			System.Console.WriteLine("Número: " + numero);

			if(numero % 2 == 0) 
			{
				System.Console.WriteLine("-----------------------");
				continue;
			}

			soma += numero;
		
			System.Console.WriteLine("Soma: " + soma);
			System.Console.WriteLine("-----------------------");

			if(soma > 9) 
			{
				System.Console.WriteLine("Você ganhou com " + i + " lançamentos");
				break;
			}
		}
	}
}

File: rafael/controle-de-fluxo/JogoDeDado2.java
class JogoDeDado2 {
	public static void main(String[] args) {
		int soma = 0;
		for(int i = 1; i <= 5; i++) {
			System.out.println("Lançamento: " + i);
			int numero = (int)(Math.random() * 6 + 1);

			System.out.println("Número: " + numero);

			if(numero % 2 == 0) {
				System.out.println("-----------------------");
				continue;
			}

			soma += numero;
		
			System.out.println("Soma: " + soma);
			System.out.println("-----------------------");

			if(soma > 9) {
				System.out.println("Você ganhou com " + i + " lançamentos");
				break;
			}
		}
	}
}
